src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -325,6 +350,154 @@ export default class TableComponent extends ComponentBase implements IComponent,
 		}
 	}
 
+	private registerRowDetailPlugin(settings: azdata.RowDetailsOptions) {
+		const rowDetail = new RowDetailView({
+			cssClass: '_detail_selector',
+			process: (item) => {
+				(<any>rowDetail).onAsyncResponse.notify({
+					'itemDetail': item,
+				}, undefined, this);
+			},
+			useRowClick: true,
+			panelRows: settings.panelRows,
+			postTemplate: (itemDetail) => itemDetail.__detailsData__,
+			preTemplate: () => '',
+			loadOnce: true,
+			detailsHtml: settings.detailsHtml
+		});
+		this._rowDetail = rowDetail;
+		this._table.registerPlugin(<any>this._rowDetail);
+	}
+
+	private registerFilterPlugin() {
+		const filterPlugin = new HeaderFilter<Slick.SlickData>();
+		this._register(attachButtonStyler(filterPlugin, this.themeService));
+		this._filterPlugin = filterPlugin;
+		this._filterPlugin.onFilterApplied.subscribe((e, args) => {
+			let filterValues = args.column.filterValues;
+			if (filterValues) {
+				this._tableData.refresh();
+				this._table.grid.resetActiveCell();
+			}
+		});
+		//this.filterPlugin.onCommand.subscribe((e, args: any) => {
```

Comment:
remove

Reply:
I left this here since as a remained that I need to migrate sorting functionality as well. But I agree, it's better to avoid commented code, so I removed this.